config/acu-strict.conf

```diff
@@ -285,6 +285,7 @@ self-call-recursive:			skip
 record-contains-depending-clause:	unconformable
 defaultbyte:				" "
 picture-l:				unconformable
+odo-in-redefines:			warning
```

Comment:
the warning is only raised when compiled with "cautions enabled" - so it should be "ok" here (and a possible cobc_acu wrapper will take the necessary `-a`  parameter and translate it which will include `-fodo-in-redefines=warning`)

... but for larger-redefines it should actually be `warning` - please adjust that when changing this file